### DISCUSSION TITLE
WIKI ZIM

### DISCUSSION THREAD

**Discussion by mikelgiles** [community/community#146875] [Programming Help]
### Select Topic Area

Question

### Body

where can I find info on ZIM software?

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @mikelgiles , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. GitHub code search lets you rapidly search, navigate and understand your code, I've gone ahead and moved it to Programing Help a more suitable category for you. Good luck!

**Comment by mecodeatlas** [comment]
Hello, @mikelgiles!

To help you, I did a search and found the [Zim's official Wiki](https://zim-wiki.org/) that provides detailed information about the Zim Software. Additionally, I recommend checking out [this forum](https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/categories/q-a) for further insights into Zim. I hope this helps! :)
